modules/ROOT/pages/workday/workday-connector-studio.adoc

```diff
@@ -32,35 +31,35 @@ To configure a connector in Anypoint Studio:
 
 == Configure an Input Source
 
-Configure an input source for the connector such as an operation for
-the Workday connector, *HTTP Listener*, or *Scheduler*.
-
-You can put the following operation for the connector in
-the *Source* area of the Studio canvas:
+Configure an input source for the connector.  Possible input source operations are:
 
-New objects trigger
+* *On New Objects* - Start your app when an object is created in Workday. 
+* *HTTP Listener* - Start your app using a browser or a command such as `curl`.
+* *Scheduler* - Start your app at timed intervals.
 
-. Click the Trigger card and select *Workday Connector*: +
-A list of object types that you can use to trigger Mule applications with the Workday connector is displayed. +
-In the Workday connector, the input source trigger is named *New Objects Trigger*.  +
-//. Specify the Workday https://community.workday.com/sites/default/files/file-hosting/productionapi/operations/index.html[Operation].
-. Select *Click here to set it up* to set the required fields for the *User Name*, *Password*,
-and *Tenant Name*. This information is available from your Workday account.
+In Studio, drag the input operation you choose from the *Mule Palette* to the *Source* panel in the canvas. 
 
-== Configure the Connector
+The required fields for the *On New Objects* input source are:
 
 [%header,cols="30s,70a"]
 |===
```

Comment:
None of the following Description entries should end with a period; none is a complete sentence